resources/certificate.rb

```diff
@@ -0,0 +1,28 @@
+resource_name :certificate
+default_action :install
```

Comment:
If there is only one action, we don't need to indicate a default. If we decide to add more actions later, then we'll need to add it. I would probably remove it until then, but it's up to you.

Reply:
done